rust/src/lib/net_state.rs

```diff
@@ -325,21 +322,20 @@ pub(crate) struct MergedNetworkState {
     pub(crate) ovsdb: MergedOvsDbGlobalConfig,
     pub(crate) routes: MergedRoutes,
     pub(crate) rules: MergedRouteRules,
-    pub(crate) memory_only: bool,
+    pub(crate) option: NetworkStateOption,
```

Comment:
If only `memory_only` is applicable to `MergedNetworkState`, we shouldn't include the whole `NetworkStateOption`. It will become confusing to know where you have to set or read `no_verify` and others. From `NetworkState` or from `MergedNetworkState`?

(Actually, now that I look at it, it is a bit weird that they are in `NetworkState` too. They are not part of the network state. They are options about how to apply a new network state and they make sense for a **desired** network state, but what do they mean for a **current** network state? I'm just thinking about it, I'm not saying that we should change it now).